console/src/components/WelcomePopup/WelcomePopup.module.less

```diff
@@ -20,9 +24,9 @@
 			font-size: 150%;
 		}
 
-		.panel .body {
-			padding: 0;
-		}
+		//.panel .body {
+		//	padding: 0;
+		//}
```

Comment:
Do we need to keep these commented codes? If it's not needed, please remove them.